tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -430,6 +430,11 @@ public static partial class Tags
         /// </summary>
         internal const string AwsQueueUrl = "aws.queue.url";
 
+        /// <summary>
+        /// The event bus name associated with the span.
+        /// </summary>
+        internal const string EventBusName = "eventbusname";
```

Comment:
Just checking if this tag name is this right. I'm wondering if it should be `event.bus.name`? `eventbusname` just seems strange to me :shrug:

Reply:
Yes, I also thought it was a bit strange, but I was following the format of 
```
internal const string TopicName = "topicname";
```

where it was all in one word.

I'm happy to change it to `event.bus.name`, what do you think?

Reply:
This should use whatever we have in other runtimes, as those tags have been checked and modified recently. I'll check what we have in JS.

Leaving [this RFC](https://docs.google.com/document/d/1450pjyhzqHTTtEvPyZGzH1cKpca7gOwtxgel_LUsR4w/edit#heading=h.w9tpabmagxvr) as it should contain some guidance

Reply:
I also saw the `rulename` https://github.com/DataDog/dd-trace-js/blob/c700341689f44b5991d36c68967a21151255866d/packages/datadog-plugin-aws-sdk/src/services/eventbridge.js#L5-L6 here, we might want to circle back with Harv @nhulston?

And check the ones we have in the Agent?

Reply:
So based on that JavaScript code, it looks like we're using the tags `resource.name`, `aws.eventbridge.source`, and `rulename`.

I will make sure to include these tags in .NET/Java/Go. However, I think we should also have a `eventbusname` tag (following the naming convention from `rulename`). But this will require me to make a change to JS/Python.

I'll check with Harv. 

Also, it looks like the Python tracer might be mixing up rule name and event bus name; those are two different fields: https://github.com/DataDog/dd-trace-py/blob/d8c8fa9a71dba6da02b1772456c9c4460dd6ca42/tests/contrib/botocore/test.py#L1837

Reply:
UPDATE: it looks like the Python/JavaScript are mixing up rulename and eventbusname; those are two different fields: https://github.com/DataDog/dd-trace-py/blob/d8c8fa9a71dba6da02b1772456c9c4460dd6ca42/tests/contrib/botocore/test.py#L1837

There's no way to get the rulename from the AWS API for PutEventsRequest in _any_ runtime, since the request takes in the busname, but not the rule name. (e.g. [JavaScript](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/eventbridge/command/PutEventsCommand/), [Java](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/eventbridge/model/PutEventsRequestEntry.html), [.NET](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/EventBridge/TPutEventsRequestEntry.html))

I talked to Harv, and we're going to keep it consistent and set the tag "rulename" to the busname. Logging a ticket to rename the keys from `rulename` to `eventbusname` across all runtimes in the future

Reply:
Cool thanks for looking into this!

Reply:
Well if we are going to rename `rulename` to `eventbusname` in the future maybe we just do it now?

Reply:
Thanks for looking into this, please do log a ticket around the changes as this might be a breaking change we want to do

Reply:
Created under [SVLS-5757]